applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -71,9 +71,19 @@ def GetDefaultParameters(cls):
             },
             "bfecc_convection" : false,
             "bfecc_number_substeps" : 10,
-            "distance_reinitialization" : "variational",
-            "distance_smoothing" : false,
-            "distance_smoothing_coefficient" : 1.0,
+            "levelset_convection_settings": {
+                "levelset_variable_name" : "",
+                "levelset_convection_variable_name" : "",
+                "levelset_gradient_variable_name" : "",
+                "max_CFL" : 1.0,
+                "max_substeps" : 0,
+                "levelset_splitting" : false,
+                "eulerian_error_compensation" : true,
```

Comment:
```suggestion
                "eulerian_error_compensation" : false,
```

Note that this implies a behavior change with respect to the current one. Aside of the fact that it would affect the existent test results, as far as I know it might have an impact on the computational efficiency.

Having these two things into consideration, we must keep the old one as default.